lib/rubocop/cop/offense_message_annotater.rb

```diff
@@ -0,0 +1,68 @@
+# frozen_string_literal: true
+
+module RuboCop
+  module Cop
+    # A annotater that takes the basic message from a cop
+    # and annotates it for output based on options or settings.
```

Comment:
Ideally the class documentation should extended a bit and yard doc should be added to the public methods.